components/script/task_queue.rs

```diff
@@ -151,11 +152,25 @@ impl<T: QueuedTaskConversion> TaskQueue<T> {
         }
     }
 
+    // Time an iteration of the event-loop.
+    fn time_event_loop(&self) {
```

Comment:
This seems to actually be a duplicate of https://github.com/servo/servo/blob/d34403047e806fa6c8c2468946f64429622ec434/components/script/script_thread.rs#L1249

I think I'll remove it...